app/assets/stylesheets/layouts/user_header.scss

```diff
@@ -0,0 +1,25 @@
+.user-profile-buttons button {
+  float: right;
+}
+
+#user-profile.container {
```

Comment:
Can we avoid using id selectors in css? You shouldn't need to you `id` unless you are going to tie this with a specific javascript behavior or you want to do anchoring